src/disclosures/js/models/school-model.js

```diff
@@ -18,7 +18,8 @@ var schoolModel = {
   processAPIData: function( values ) {
     values.jobRate = values.jobRate || '';
     values.programLength /= 12;
-    values.medianSalary = values.salary || values.medianAnnualPay;
+    values.medianSalary = values.salary || values.medianAnnualPay ||
```

Comment:
All the other lines in this sections seem to be first looking for program level data, then school level data. Can you talk a little more about what's now going on for the medianSalary value, or point me at the documentation about it? The variable names are so similar, and it's hard to know where each datapoint is coming from or represents. 😕